conjureup/juju.py

```diff
@@ -29,7 +29,7 @@
 def requires_login(f):
     def _decorator(*args, **kwargs):
         if not this.IS_AUTHENTICATED:
-            login()
+            login(True)
```

Comment:
It's more readable to put the keyword in here, like `login(force=True)` , then you know what the True is for